NinchatSDKSwift/Implementations/Models/Queue.swift

```diff
@@ -6,8 +6,19 @@
 
 import Foundation
 
+enum QueuePermissionType: String {
+    case member
+    case agent
```

Comment:
The "agent" option is deprecated for queue attributes. It's not even defined for channel attributes.

You can just leave that out, since currently the only relevant one for the SDK is "member".

Reply:
Yes, but since I still get both in queue's attributes, I have to add both options to the enum to correctly model the response. I already use the "member" type

Reply:
Well, that's not an exhaustive list. And more options may be added to the API at any time. Your model should handle unknown options without breaking.

Reply:
(Same goes for all enums returned by the API.)

Reply:
In that case, the function does not break since it returns null which doesn't break the logic: [https://github.com/somia/ninchat-sdk-ios-swift/blob/1f3fff1e3db2a63d437f853c6e37632eecbb253d/NinchatSDKSwift/Implementations/Extensions/NINLowLevelClientProps%2BExtension.swift#L174]()

Reply:
Ok. You could just define the options that are actually relevant to the SDK, and handle null as _so-it-is-some-unknown-option-value-which-means-that-I-cannot-use-this-feature_.

Or you could define all of the options defined in the API spec, but then your moral responsibility would be to update the model whenever the API is expanded. :smile: (I'm actually planning on adding a new upload option...)

Reply:
Yes, that makes sense. I'll update the enum in a commit